static/openapi/est-spec.json

```diff
@@ -0,0 +1,1543 @@
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/MetricsResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid date parameters"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        }
+      }
+    },
+    "/api/est/v1/activity/monthly": {
+      "get": {
+        "tags": [
+          "Activity Metrics"
+        ],
+        "description": "Get monthly activity metrics based on start and end date-time parameters",
+        "parameters": [
+          {
+            "in": "header",
+            "name": "X-PhotonIQ-CustomerID",
+            "schema": {
+              "type": "string"
+            },
+            "required": true,
+            "description": "PhotonIQEST CustomerID"
+          },
+          {
+            "in": "query",
+            "name": "from",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "required": true,
+            "description": "The UTC start date and time when activity information is retrieved from (e.g. 2022-01-01T00:00:00Z)"
+          },
+          {
+            "in": "query",
+            "name": "to",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "description": "The UTC end date and time when activity information is retrieved from (e.g. 2022-01-01T00:00:00Z)",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Activity metrics retrieved successfully",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/MetricsResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid date parameters"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        }
+      }
+    },
+    "/api/est/v1/change-history": {
+      "get": {
+        "tags": [
+          "Change History"
+        ],
+        "description": "Get the change history of PhotoniqEST service",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "from",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "description": "Optional. Filter results from a specified date and time in UTC (e.g. 2022-01-01T00:00:00Z)."
+          },
+          {
+            "in": "query",
+            "name": "to",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "description": "Optional. Filter results up to a specified date and time in UTC (e.g. 2022-01-01T00:00:00Z)."
```

Comment:
If this is optional, what will the query do?  Does it default to something?  Or just not use this date in its query?

Reply:
@dlozina-macrometa, please make a note of this for next rev.